src/components/courseTimeline.tsx

```diff
@@ -0,0 +1,57 @@
+const CourseTimeline = () => {
+  return (
+    <div className="mx-auto flex w-[140vh] flex-col items-center justify-start gap-[2vh] rounded-xl bg-[#023851] px-[12vh] py-[6vh] font-malayalam text-[#FFF4B8]">
```

Comment:
```suggestion
    <div className="flex w-3/4 flex-col items-center rounded-xl bg-[#023851] px-[12vh] py-[6vh] font-malayalam text-[#FFF4B8]">
```
also add the colors within tailwind config file